### ISSUE TITLE
Show some info on AMO

### ISSUE THREAD

**Issue by MikkCZ** [diox/firefox-minimap-scroller#19]
On AMO content scripts are blocked by Firefox. Can you please add some short message to the sidebar, like "Not available for this site" so it does not look broken?

**Comment by Gitoffthelawn** [comment]
Unless, of course, the browser is configured to allow content scripts on AMO.

**Comment by diox** [comment]
I'll have to investigate, I'm not sure if the sidebar receives an error when trying to connect to the content script on AMO/about: pages.

**Comment by Gitoffthelawn** [comment]
@MikkCZ If you like, try:
/* Allow extensions to interact with pages such as addons.mozilla.org */
user_pref("privacy.resistFingerprinting.block_mozAddonManager", true);

**Comment by MikkCZ** [comment]
Closing (as won't fix) given the add-on development seems stopped anyway.
